.gometalinter.json

```diff
@@ -9,9 +9,9 @@
     "Exclude": [
         "\/src\/(crypto|internal|net|os|runtime|syscall)\/",
         "^pkg\/nodebootstrap\/assets.go",
-        "^pkg\/testutils\/ginkgo.go",
-        "^pkg\/testutils\/client.go",
+        "^pkg\/testutils\/(ginkgo|client).go",
         "^pkg\/testutils\/mockprovider\/mock_provider.go",
+        "^pkg\/cfn\/template\/matchers\/matchers.go",
```

Comment:
I don't see the point of adding copy-pasted documentation to all functions here, if someone knows how to exclude this file from godoc check separately, pleaser let me know.